tools/make/image.mk

```diff
@@ -67,6 +64,27 @@ image.push.%:
 	@echo "===========> Pushing docker image tag $(IMAGE):$(TAG) for $(ARCH)"; \
 	$(DOCKER) push $(IMAGE):$(TAG); \
 
+.PHONY: image.multiarch.verify
+image.multiarch.verify:
+	$(eval pass := $(shell ))
+	@docker buildx --help | grep -qw "docker buildx" || { \
+		echo "Cannot find `docker buildx`, please install first"; \
+		exit 1; }
+
+.PHONY: image.multiarch.setup
+image.multiarch.setup: image.verify image.multiarch.verify
+	@docker run --rm --privileged tonistiigi/binfmt --install all # Install QEMU emulators
```

Comment:
it is safer to only install emulators for the interested platforms `--platform "${BUILDX_PLATFORMS}`

Reply:
Sure, let me fix it.